lib/matplotlib/artist.py

```diff
@@ -85,7 +85,7 @@ def __init__(self):
         self._contains = None
         self._rasterized = None
         self._agg_filter = None
-        self._mouseover = False
+        self._mouseover = type(self).get_cursor_data != Artist.get_cursor_data
```

Comment:
IMHO, this would benefit from a comment.

Reply:
sounds fair